sdk/core/core/src/az_http_response.c

```diff
@@ -44,7 +44,7 @@ AZ_NODISCARD az_result _az_get_digit(az_span* self, uint8_t* save_here)
   uint8_t c_ptr = az_span_ptr(*self)[0];
   if (!isdigit(c_ptr))
   {
-    return AZ_ERROR_PARSER_UNEXPECTED_CHAR;
```

Comment:
Lets not change these.  The specific parser error if more informative

Reply:
Done.